alpha/apps/kaltura/lib/reports/myReportsMgr.class.php

```diff
@@ -1312,7 +1312,7 @@ private static function executeQuery ( $query )
 		
 		$error_function = $mysql_function.'_error';
 		if (!$db_selected) {
-			throw new kCoreException('Can\'t use foo : ' . $error_function($link), kCoreException::INVALID_QUERY);
+			throw new kCoreException("mysqli_select_db($link , ".$db_config["db_name"].") failed." . $error_function($link), kCoreException::INVALID_QUERY);
```

Comment:
I don't think it makes sense to print the $link, also, please add a white space after the dot and before the output of $error_function.